docs/1.getting-started/6.data-fetching.md

```diff
@@ -301,7 +301,7 @@ const { data } = await useFetch('/api/me', { headers })
 ```
 
 ::alert{type="warning"}
-Be very careful before proxying headers to an external API and just include headers that you need. Not all headers are safe to be bypassed and might introduce unwanted behavior. Here is a list of common headers that are NOT to be proxied:
+Practice extreme caution when proxying headers to an external API, and only include headers you need. Not all headers are safe to be bypassed and might introduce unwanted behavior. Here is a list of common headers that are NOT to be proxied:
```

Comment:
What about _Use extreme caution_ instead? The former is pretty rare.
```suggestion
Practice extreme caution when proxying headers to an external API, and only include headers you need. Not all headers are safe to be bypassed and might introduce unwanted behavior. Here is a list of common headers that **shouldn't** be proxied:
```

Reply:
Use extreme caution sounds a bit odd to me for some reason, but I'm open to opinions!

Reply:
Added!